repository/t.json

```diff
@@ -2170,26 +2170,27 @@
 			]
 		},
 		{
-			"name": "Tmux Syntax Highlight",
-			"previous_names": ["Tmux"],
-			"details": "https://github.com/keqh/sublime-tmux-syntax-highlight",
-			"labels": ["language syntax"],
+			"name": "tmux",
+			"previous_names": ["tmux_open"],
+			"details": "https://github.com/huntie/sublime-tmux",
+			"labels": ["tmux", "terminal"],
 			"releases": [
 				{
-					"sublime_text": "*",
-					"branch": "master"
+					"sublime_text": ">=3000",
+					"platforms": ["linux", "osx"],
+					"tags": true
 				}
 			]
 		},
 		{
-			"name": "tmux_open",
-			"details": "https://github.com/huntie/sublime-tmux",
-			"labels": ["tmux", "terminal"],
+			"name": "Tmux Syntax Highlight",
+			"previous_names": ["Tmux"],
```

Comment:
I'm not sure how Package Control handles this on case-insensitive file systems.
Imo it would be better to remove the `previous_names` entry for the old package to not potentially cause issues here.

Reply:
Sure thing, this has been rebased.